include/highfive/bits/H5Easy_Eigen.hpp

```diff
@@ -0,0 +1,190 @@
+/*
+ *  Copyright (c), 2017, Adrien Devresse <adrien.devresse@epfl.ch>
+ *
+ *  Distributed under the Boost Software License, Version 1.0.
+ *    (See accompanying file LICENSE_1_0.txt or copy at
+ *          http://www.boost.org/LICENSE_1_0.txt)
+ *
+ */
+#ifndef H5EASY_BITS_EIGEN_HPP
+#define H5EASY_BITS_EIGEN_HPP
+
+#include "../H5Easy.hpp"
+#include "H5Easy_misc.hpp"
+
+#ifdef HIGHFIVE_EIGEN
+
+namespace HighFive
```

Comment:
Please follow coding style of existing code base, i.e:

```cpp
namespace A {
namespace B {

foo my_function() {
  instruction1; // no extra newline between every instruction
  if (test) {
    instruction2;
  }
}

} // namespace B
} // namespace A
```

Reply:
We have the `.clang-format` file in the root, just apply the style.

Reply:
@ferdonline Cool, do I have to run `clang-format -style=file include/highfive/H5Easy.hpp` from the project's root?

Reply:
`-style=file` is redondant. `clang-format` looks for `.clang-format` in parent directories if not found where source is, so it can be run from anywhere within project directory.

use option `-i` option to inplace edit the file.